src/components/TaxServiceModal/index.tsx

```diff
@@ -0,0 +1,129 @@
+import { Trans } from '@lingui/macro'
+import { ButtonEmphasis } from 'components/Button'
+import { ButtonSize, ThemeButton } from 'components/Button'
+import { Box } from 'nft/components/Box'
+import { bodySmall, subhead } from 'nft/css/common.css'
+import { memo } from 'react'
+import styled from 'styled-components/macro'
+
+import Modal from '../Modal'
+import CointrackerFullLogo from './CointrackerFullLogo.png'
+import { StyledXButton } from './TaxServiceBanner'
+import TokenTaxFullLogo from './TokenTaxFullLogo.png'
+
+interface TaxServiceModalProps {
+  isOpen: boolean
+  onDismiss: () => void
+}
+
+interface TaxServiceOptionProps {
+  logo: any
+  description: string
+  url: string
+}
+
+const InnerContainer = styled.div`
+  background-color: ${({ theme }) => theme.backgroundSurface};
+  overflow: hidden;
+  display: flex;
+  width: 420px;
+  height: 268px;
+  flex-direction: column;
+  position: relative;
+  gap: 20px;
+  padding: 16px;
+`
+
+const TaxOptionContainer = styled.div`
+  display: flex;
+  flex: 1;
+  gap: 16px;
+  justify-content: center;
+`
+
+const TaxOptionDescription = styled.div`
+  display: flex;
+  height: 100%;
+  justify-content: center;
+  user-select: none;
+  text-align: center;
+`
+
+const TaxOption = styled.div`
+  align-items: center;
+  background-color: ${({ theme }) => theme.backgroundModule};
+  border-radius: 12px;
+  cursor: auto;
+  display: flex;
+  flex-direction: column;
+  flex: 1;
+  justify-content: space-between;
+  padding: 12px;
+  gap: 16px;
+`
+
+const StyledImageContainer = styled(Box)`
+  width: 75%;
+  height: 80%;
+  cursor: auto;
+  object-fit: contain;
+`
+
+const Button = styled(ThemeButton)`
+  cursor: pointer;
+  width: 100%;
+  margin-right: auto;
+`
+
+const TOKEN_TAX_URL = 'https://tokentax.co/uniswap?via=uniswap'
+const COINTRACKER_URL = 'https://www.cointracker.io/partner/uniswap?utm_source=uniswap'
+
+const TOKEN_TAX_DESCRIPTION = 'Save 10% on all plans'
+const COINTRACKER_DESCRIPTION = 'New and existing users save up to 20%'
+
+function TaxServiceOption({ description, logo, url }: TaxServiceOptionProps) {
+  const openTaxServiceLink = () => {
+    window.open(url, '_blank')
+  }
+
+  return (
+    <TaxOption tabIndex={0}>
```

Comment:
should `tabIndex` be a prop so the parent can set them as `0` and `1` ? is this for a11y or something else?

Reply:
i don't think we need to set as a prop, setting it to 0 is our only use case here.